core/main.go

```diff
@@ -167,9 +188,17 @@ func main() {
 	logrus.Infof("starting MESG Engine version %s", version.Version)
 
 	go func() {
-		if err := server.Serve(dep.config.Server.Address); err != nil {
+		if err := server.Serve(cfg.Server.Address); err != nil {
+			logrus.Fatalln(err)
+		}
+	}()
+
+	logrus.WithField("seeds", cfg.Tendermint.P2P.Seeds).Info("starting tendermint node")
+	go func() {
+		if err := node.Start(); err != nil {
 			logrus.Fatalln(err)
 		}
+		select {}
```

Comment:
why this select?

Reply:
node.Start doesn't block like similar http/grpc method server. so, in fact, it dosesn't need to be started it in a separate goroutine

Reply:
@krhubert please improve